gql/sort.go

```diff
@@ -94,7 +94,11 @@ func unprotectedListSort(params *sortParameters, list []interface{}, errChan cha
 
 	var less lessFunc
 	extracFunc := func(index int) interface{} {
-		return DeepExtractField(list[index], params.field)
+		value, err := DeepExtractField(list[index], params.field)
+		if err != nil {
+			panic(err)
```

Comment:
yuk to the entire function not just your addition. Though I'm not sure we have an alternative.

Reply:
Agreed... wish it was better